src/reducers/noteReducer.js

```diff
@@ -1,12 +1,20 @@
-export const noteReducer = (state = [], action ) => {
+export const noteReducer = (state = [], action) => {
   switch (action.type) {
-    case 'ADD_NOTE':
+    case "ADD_NOTE":
       return [...state, action.note];
-    case 'DELETE_CARD':
-      return  state.filter(note => note.id != action.id);;
-    case 'GET_ALL_NOTES':
-      return action.notes
+    case "DELETE_CARD":
```

Comment:
It's a shame that we have this naming issue here.  We might want to rename some things here so that our reducer doesn't have 2 types of nouns in the cases.